etc/kayobe/dnf.yml

```diff
@@ -8,6 +8,8 @@
 # dnf_config:
 #   proxy: http://proxy.example.com
 #dnf_config:
+dnf_config:
+  install_weak_deps: False
```

Comment:
Why is this required?

Reply:
It's to keep hosts free of crap like PackageKit